packages/cli/lib/project-generator.js

```diff
@@ -210,7 +212,7 @@ module.exports = class ProjectGenerator extends BaseGenerator {
   scaffold() {
     if (this.shouldExit()) return false;
 
-    this.destinationRoot(this.projectInfo.outdir);
+    this.destinationRoot(utils.untildify(this.projectInfo.outdir));
```

Comment:
If you add `utils.untildify` as a `filter` to the prompt for `outdir` then this wouldn't be necessary.

Doing it there instead of here would scope these changes squarely in to the user-input processing instead of also changing some of the scaffolding logic.

Reply:
If you want to get really fancy, you might even be able to add a `transformer: tildify`. If I understand the [docs](https://www.npmjs.com/package/inquirer#question), this would make it so if the user inputs `/Users/me/work/myNewApp` then the answer will be shorted to `~/work/myNewApp` when it is printed back to the user but it will be _stored_ as the absolute path.

Reply:
yeah I was doing the opposite way: stored the shorted path and generate the files by absolute path D: 
I guess the way you suggested is better as absolute paths are more reliable.

Reply:
I think this line can be reverted back, since you are calling `untildify` on L171 above. PTAL.